### PR TITLE
Disable slow metadata.

### PR DESCRIPTION
**Pull request by fabricematrat**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2268/

File: internal/charmstore/server.go [hunk 1]
```diff
@@ -150,6 +150,11 @@ type ServerParams struct {
 	// DockerRegistryTokenDuration is the time a docker registry
 	// token will be valid for after it is created.
 	DockerRegistryTokenDuration time.Duration
+
+	// DisableSlowMetadata indicates that we will have a placeholder response instead of a full
+	// response for slow enpoints.
```
Comment (mhilton) [inline internal/charmstore/server.go]: endpoints

File: internal/charmstore/server.go [hunk 2]
```diff
@@ -150,6 +150,11 @@ type ServerParams struct {
...
+	// This is temporary.
```
Comment (mhilton) [inline internal/charmstore/server.go:156]: Why is it temporary and how is it going to be changed in future.

**Review by mhilton — COMMENTED** [review]
Shouldn't this also cover the slow charm-revisions too? although I suspect at least that one is actuallyl being used. I don't see how we can selectively disable the slower bits though.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2272/

**Review by rogpeppe — APPROVED** [review]
LGTM thanks

**Review by mhilton — APPROVED** [review]
LGTM

**Comment by fabricematrat** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-merge

**Pull request merged**